orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Execution.java

```diff
@@ -192,6 +192,20 @@ public void setEndTime(@Nullable Long endTime) {
     this.endTime = endTime;
   }
 
+  /**
+   * Gets the start ttl timestamp for this execution. If the execution has not
+   * started before this timestamp, the execution will immediately terminate.
+   */
+  private Long startTimeTtl;
+
+  public @Nullable Long getStartTimeTtl() {
```

Comment:
What unit is this in? Comment suggests it's a timestamp but is it actually a duration? Could we use `java.time.Instant` or `java.time.Duration` as appropriate?